src/ranged.cpp

```diff
@@ -530,19 +530,6 @@ int player::fire_gun( const tripoint &target, int shots, item& gun )
         debugmsg( "Attempted to fire zero or negative shots using %s", gun.tname().c_str() );
     }
 
-    const skill_id skill_used = gun.gun_skill();
```

Comment:
What's going to train the specific gun skills (handguns, rifles, etc) with this gone?